addons/website_payment/static/tests/tours/donation.js

```diff
@@ -51,6 +52,11 @@ registerWebsitePreviewTour(
             content: "Check if custom amount radio input has value 55",
             trigger: ':iframe input#other_amount[value="55.0"]',
         },
+        {
+            content: "Click on custom amount value",
+            trigger: ":iframe input#other_amount_value",
+            run: "click",
+        },
```

Comment:
What is the use of this step here? we are clicking on Other amount input and then in next step we are clicking on input with value 25.